modules/geo-layers/src/h3-layers/h3-hexagon-layer.js

```diff
@@ -111,35 +111,44 @@ export default class H3HexagonLayer extends CompositeLayer {
     const {
       data,
       getHexagon,
-      updateTriggers,
-
+      wireframe,
       elevationScale,
-      extruded,
       fp64,
-
+      material,
+      extruded,
+      stroked,
+      lineWidthScale,
+      lineWidthMinPixels,
+      lineWidthMaxPixels,
       getColor,
       getElevation,
-      material
+      getLineColor,
+      getLineWidth,
+      updateTriggers
     } = this.props;
 
-    const SubLayerClass = this.getSubLayerClass('hexagon-cell-hifi', SolidPolygonLayer);
+    const SubLayerClass = this.getSubLayerClass('hexagon-cell-hifi', PolygonLayer);
 
     return new SubLayerClass(
       {
         filled: true,
         elevationScale,
         extruded,
         fp64,
-        getFillColor: getColor,
+        wireframe,
+        stroked,
+        lineWidthScale,
+        lineWidthMinPixels,
+        lineWidthMaxPixels,
+        material,
         getElevation,
-        material
+        getFillColor: getColor,
+        getLineColor,
+        getLineWidth
       },
       this.getSubLayerProps({
         id: 'hexagon-cell-hifi',
-        updateTriggers: {
-          getFillColor: updateTriggers.getFillColor,
-          getElevation: updateTriggers.getElevation
-        }
+        updateTriggers
```

Comment:
The existing code had a bug.

The user-provided `updateTriggers` uses keys that are consistent with this layer's accessors. Therefore, you need to manually map accessor names of the H3HexagonLayer API to those of the PolygonLayer API. e.g.

- `updateTriggers.getColor` to `updateTriggers.getFillColor`
- `updateTriggers.getHexagon` to `updateTriggers.getPolygon`